### DISCUSSION TITLE
CapibaraZero 0.5.1

### DISCUSSION THREAD

**Discussion by andreock** [CapibaraZero/fw#52] [Announcements]
Changelog:

- BadUSB: Add support to ATTACKMODE(HID or MSC)
- NFC: Add write and format feature for MIFARE tags
- GUI: Use hardware SPI
- SubGHZ: Add support for CC1101
- File browser: Make page scrollable
- Bump ArduinoJson to 7 and adapt code to it
- Support LilyGo T-Embed CC1101
- Fix various crash

PCB Version: 1.1.2

Flash using:
python3 esptool.py --chip esp32s3 --port UPLOAD_PORT --baud 460800 --before default_reset --after hard_reset write_flash -z --flash_mode dio --flash_freq 80m --flash_size detect 0x0000 bootloader.bin 0x8000 partitions.bin 0xe000 boot_app0.bin 0x10000 firmware.bin

Blog page: https://capibarazero.com/blog/capibarazero-0-5-1-out

<hr /><em>This discussion was created from the release <a href='https://github.com/CapibaraZero/fw/releases/tag/0.5.1'>CapibaraZero 0.5.1</a>.</em>

**Comment by SuthaharanTharunjan** [comment]
I like your dedication and work, guys. I think your fw doesn't have that much reach as bruce firmware has. I think it's because of user experience. bruce has a nice interface than capibara, so please put some work in Ui that would attract more people. Your firmware will gain good community support. Thanks for your time and effort. That's my thoughts about your firmware.
